controller/linodecluster_controller_helpers.go

```diff
@@ -40,3 +51,110 @@ func (r *LinodeClusterReconciler) removeMachineFromLB(ctx context.Context, logge
 	}
 	return nil
 }
+
```

Comment:
can we make this a "common" function instead on a cluster reconciler method?
The only thing it uses from the reconciler is the TracedClient. If we just pass the client, other reconcilers could use the same function like [this one](https://github.com/linode/cluster-api-provider-linode/blob/main/controller/linodemachine_controller_helpers.go#L191)

Reply:
updated to pass the traced client